products/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+
+# Create your models here.
+
+class Category(models.Model):
+    name              = models.CharField(max_length=40)
+
+    class Meta:
+        db_table = 'categories'
+
+class SubCategory(models.Model):
+    name         = models.CharField(max_length=40)
+    category     = models.ForeignKey('Category',on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'sub_categories'
+
+class Product(models.Model):
+    name         = models.CharField(max_length=40)
+    price        = models.DecimalField(max_digits=8, decimal_places=2)
+    weight       = models.CharField(max_length=10)
+    sub_name     = models.CharField(max_length=40)
+    descriptio   = models.TextField()
+    sub_category = models.ForeignKey('SubCategory',on_delete=models.CASCADE)
+    tag          = models.ManyToManyField('Tag', through='ProductTag')
```

Comment:
```suggestion
    tags          = models.ManyToManyField('Tag', through='ProductTag')
```
many to many 필드 같은 경우 복수형 단어로 해주어야 나중에 view에서 로직을 작성할 때 더 직관적이게 나타 낼 수 있습니다.
(같은 의미에서 related_name을 지정할 때도 복수로 해주시면 됩니다.)